src/Core/Settings.cpp

```diff
@@ -6917,6 +6917,9 @@ Allow experimental delta-kernel-rs implementation.
 )", BETA) \
     DECLARE(Bool, allow_experimental_insert_into_iceberg, false, R"(
 Allow to execute `insert` queries into iceberg.
+)", EXPERIMENTAL) \
+    DECLARE(Bool, write_full_path_insert_iceberg, false, R"(
```

Comment:
```suggestion
    DECLARE(Bool, write_full_path_in_iceberg_metadata, false, R"(
```
(the setting name sounded grammatically incorrect, this way seems better and more clear)